packages/web/src/js/Tooltip.ts

```diff
@@ -99,6 +146,16 @@ class Tooltip extends BaseComponent {
     );
   }
 
+  getTipTooltipWrapper() {
+    const id = this.tip.getAttribute('id');
+
+    const triggerWrapperElement = document
+      .getElementById(id!)
+      ?.closest('[data-spirit-element="tooltip-wrapper"]') as HTMLElement;
```

Comment:
You can use `[class*="TooltipWrapper"]` to match the class like a regular, e.g. get the closest class that contains `TooltipWrapper`. This should be non-breaking and also prefix/suffix prune.

Reply:
I would rather check this also with @crishpeen and @adamkudrna. And plan how we refactor this in the future. Maybe the data element attribute should be used. Or we can use the class name if we consider this safe. 🤔

Reply:
I created #1231 to take it down once and forever 🙂.

Reply:
thank you